ironfish/src/node.ts

```diff
@@ -284,6 +304,7 @@ export class IronfishNode {
     }
 
     this.telemetry.submitNodeStarted()
+    this.minedBlocksIndexer.start()
```

Comment:
The telemetry is wrong now based on this ordering right?

Reply:
Good catch, thanks!